### ISSUE TITLE
Network never predicts a ROI as background

### ISSUE THREAD

**Issue by stekau** [jwyang/faster-rcnn.pytorch#829]
I trained a Faster RCNN Network using the BDD100K Dataset. The BDD100K Dataset has several labels like car, bus or pedestrian but no background label. The training was fine and the network makes some good predictions. The only problem is, the network never labels a ROI as background. Therefore the 300 given ROIs are labled only with the classes contained in the dataset. This leads to a good but not a very good mAP as you can imagine if the network would lable some of the 300 ROIs as background the mAP would rise because the false positives would decrease.

I tried to use one of the given dataloader and changed it to use the BDD100K dataset. This dataloader also contains the variable self._classes where "__background__" is given. The amount of classes in the last layer is fine but the "__background__" class is always zero. Does anybody has an idea why the network is not learning what an background is? Is there any configuration I missed out? And does the code learn for other datasets what an background is?

I you need further information please do not hesitate and ask me! I appreciate your effort!

Thanks!
